spec/models/locu_data_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe LocuData, type: :model, vcr: true do
+  it 'successfully receives json data from locu with default parameters' do
+    locu_data = LocuData.new
+    locu_data.search_function
+    expect(locu_data.data['status']).to eq('success')
+    locu_data.data['venues'].each do |venue|
+      expect(venue['location']['locality']).to eq(IpApi.new.data['city'])
+      expect(venue['categories'].any? { |category| category['str_id'] == 'restaurants' }).to eq(true)
+    end
+  end
+
+  it 'can search by name' do
+    locu_data = LocuData.new( { 'name' => 'bistro central parc'} )
+    locu_data.search_function
+    expect(locu_data.data['venues'].count).to eq(1)
+    expect(locu_data.data['venues'][0]['locu_id']).to eq('3ac33e6306c52465e30a')
+    expect(locu_data.data['venues'][0]['name']).to eq('Bistro Central Parc')
+  end
+
+  it 'can search by open times' do
+    locu_data = LocuData.new( { 'open_hours' => { 'monday' => ['18:00'] } } )
+    t = Time.local(2016, 1, 19, 18, 0, 0)
```

Comment:
Do not use `Time.local` without zone. Use one of `Time.zone.local`, `Time.local.current`, `Time.local.in_time_zone`, `Time.local.utc`, `Time.local.getlocal`, `Time.local.iso8601`, `Time.local.jisx0301`, `Time.local.rfc3339`, `Time.local.to_i`, `Time.local.to_f` instead.